api/jreleaser-model-api/src/main/java/org/jreleaser/model/api/deploy/maven/Nexus2MavenDeployer.java

```diff
@@ -32,6 +32,8 @@ public interface Nexus2MavenDeployer extends MavenDeployer {
 
     String getSnapshotUrl();
 
+    String getSnapshotServiceUrl();
```

Comment:
Why is this needed? Could it be deduced from the snapshot url?

Reply:
```
https://oss.sonatype.org/content/repositories/snapshots/
https://oss.sonatype.org/service/local/repositories/snapshots/content/
```
they are slightly different, so my thinking here was, just to stay on the safe side, give the user an option to provide the value. Let me try running nexus 2/3 locally and see if the URL structure is predictable and if I can get the repository name (`snapshots`) and reconstruct the other url.